seed.js

```diff
@@ -0,0 +1,15 @@
+const data = require('./seedData.js');
+const db = require('./db/index.js');
+
+const randomProduct = (obj) => {
```

Comment:
If you decide to change your seedData into an array of objects per my other comment, I suggest changing your parameter here to something more fitting. Maybe 'products'.